common/models/user.js

```diff
@@ -756,10 +756,20 @@ module.exports = function(User) {
 
     UserModel.validatesFormatOf('email', { with: re, message: g.f('Must provide a valid email') });
 
-    // FIXME: We need to add support for uniqueness of composite keys in juggler
```

Comment:
composite key uniqueness is actually kind of supported, if you have a main property, u can use `scopedTo`
https://github.com/strongloop/loopback-datasource-juggler/blob/45e14af4a981fcf376276babeb344ceac578186c/lib/validations.js#L225